go/ql/lib/semmle/go/dataflow/ExternalFlow.qll

```diff
@@ -434,6 +434,10 @@ private class SummarizedCallableAdapter extends SummarizedCallable {
     exists(Provenance provenance |
       summaryElement(this, input, output, kind, provenance, model) and
       provenance.isGenerated()
+    ) and
+    not exists(Provenance provenance |
+      neutralElement(this, "summary", provenance) and
+      provenance.isManual()
```

Comment:
isGenerated

Reply:
Actually it is correct. The idea, which could do with a comment explaining it, is that generated summary models are overridden by manual neutral models. This allows us to fixed generated models in a way which will persist when they are regenerated.

Reply:
Let me know if you think there should be a comment briefly explaining this (and feel free to give a suggestion).